critiquebrainz/db/review.py

```diff
@@ -415,10 +415,11 @@ def list_reviews(*, inc_drafts=False, inc_hidden=False, entity_id=None,
                SUM(
                    CASE WHEN vote = 't' THEN 1
                    WHEN vote = 'f' THEN -1 WHEN vote IS NULL THEN 0 END
-               ) AS rating,
+               ) AS popularity,
                latest_revision.id as latest_revision_id,
                latest_revision.timestamp as latest_revision_timestamp,
                latest_revision.text as text,
+               latest_revision.rating as rating,
```

Comment:
One way to keep another pull request in sync with another unmerged branch is to keep rebasing the new work on top of it. Commits from another PR will show up there, but as long as it's made clear that the work in that PR depends on another it shouldn't be a problem.

In general I'd suggest to avoid a situation like that if it's confusing to keep dependent PRs in sync.